server.js

```diff
@@ -77,50 +75,47 @@ app.post("/signup", async (req, res) => {
     try {
```

Comment:
Not strictly related to this change, but why is `await User.findOne(...)` outside of the `try`? If anything might throw, this is it.